configure.ac

```diff
@@ -856,7 +856,7 @@ fi
 BITCOIN_QT_INIT
 
 dnl sets $bitcoin_enable_qt, $bitcoin_enable_qt_test, $bitcoin_enable_qt_dbus
-BITCOIN_QT_CONFIGURE([$use_pkgconfig], [qt5])
+BITCOIN_QT_CONFIGURE([$use_pkgconfig])
```

Comment:
missing changes for `bitcoin_qt_got_major_vers` (line 1450)

Reply:
still not fixed https://github.com/bitcoin/bitcoin/pull/13458/files#diff-67e997bcfdac55191033d57a16d1408aL1425